src/components/Request/Pay/Views/GeneralViews/Error.view.tsx

```
@@ -1,18 +1,13 @@
 'use client'
 import Icon from '@/components/Global/Icon'
 
-import { useRouter } from 'next/navigation'
 import Link from 'next/link'
 
-export const NotFoundClaimLink = () => {
-    const router = useRouter()
-
+export const Error = ({ errorMessage }: { errorMessage: string }) => {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)